platform/ui/src/components/IconButton/IconButton.jsx

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import classnames from 'classnames';
 
 const baseClasses =
-  'text-center items-center justify-center outline-none transition duration-300 ease-in-out font-bold focus:outline-none';
+  'text-center items-center justify-center outline-none font-bold focus:outline-none';
```

Comment:
I'm curious about why you removed the transition from the IconButton @igoroctaviano

Reply:
@rodrigolabs well remembered! so, at first at thought it was making the buttons feel a little leggy in the transitions but then i had trouble matching the transition of the button and the new  arrow below the button. Lets discuss about these transitions, do you feel them smooth enough?

Reply:
I think it's ok to keep the transition, since other things like the Button itself have the same transition, to keep the consistency. I can revert it in my PR. And then later we can try to do this minor adjustment in the arrow below the button. Is that ok for you?